Adding Company to InnerSource in Action

clcoffey opened the pull request:
Adding a new logo and company to the InnerSource in Action Page.

rrrutledge commented:
Awesome work!

Is it possible to see this change deployed to _GitHub_ pages before merge so as to very that things still work?

lenucksi commented:
> Awesome work!
> 
> Is it possible to see this change deployed to _GitHub_ pages before merge so as to very that things still work?

The changes built on CI which means that Jekyll didn't crash and burn. 
If the site looks the same is something that would need to be validated by yourself locally, with a stage deployment or some Selenium excess.

dellagustin-sap commented:
@rrrutledge that could be done manually, but I would be the equivalent of merging it and letting the build pipeline deploy it.

Reviewing it locally and having exactly the same experience you have when you deploy to the productive website is virtually impossible.

We could try to increase test coverage, we would have to be very smart about it, not to commit to stuff we are not sure we want to keep, otherwise, when you refactor code, you also have to refactor test code.

lenucksi commented:
> Reviewing it locally and having exactly the same experience you have when you deploy to the productive website is virtually impossible.

I would argue that in the current state of things a manual and local preview will be sufficient in nearly all cases. It if breaks, we can go back very fast. It's not like we're operating the Google front page including search with that website in my opinion. ;)
I wouldn't complain about having an automated staging deployment though. :D

lenucksi commented:
> Reviewing it locally and having exactly the same experience you have when you deploy to the productive website is virtually impossible.

I would argue that in the current state of things a manual and local preview will be sufficient in nearly all cases. It if breaks, we can go back very fast. It's not like we're operating the Google front page including search with that website in my opinion. ;)
I wouldn't complain about having an automated staging deployment though. :D

dellagustin commented:
@lenucksi I am of the very same opinion.

rrrutledge commented:
Sounds good.  Just check the altered links after deployment?

lenucksi commented:
When should we merge this? Once all the checkboxes are ticked?

lenucksi commented:
> @lenucksi
> 
> > When should we merge this? Once all the checkboxes are ticked?

As soon as we can, i.e. as soon as the merge-conflict that exists currently is handled.

> Good question, what happens if you merge now and I push another commit to the branch that originated this PR?

A cute baby kitten dies. ;)

> Would it be reopened? Is that an acceptable GitHub workflow? I never did it like this, but it "smells bad".

No, I don't think so. No. Yes it does in my book too. ;)

> 
> Maybe I should create an issue with the title "Refactor: reduce usage of absolute paths", transfer the checkboxes there and we go merging bite size PRs (as this one) with each individual correction.
> 
> Any opinions?

Sounds fine to me. Please close this PR if you're going to split it the things up. If you reference this PR in the new PRs that would be very useful :D
I'm all for small PRs. Most of the times.

clcoffey committed:
Adding Company to InnerSource in Action

Adding a new logo and company to the InnerSource in Action Page.

clcoffey committed:
Changing the Europace Logo

The Europace Logo was representing the wrong company. I changed it to the right one.

voborgus commented:
thanks! sorry for the long merging
